packages/workspace/src/utils/buildale-libs-utils.ts

```diff
@@ -103,7 +103,14 @@ export function updatePaths(
 ) {
   dependencies.forEach(depp => {
     if (isBuildable(depp.node)) {
-      paths[depp.name] = [getOutputPath(depp.node)];
+      const out = getOutputPath(depp.node);
+      for (const path of Object.keys(paths)) {
+        if (path === depp.name) {
+          paths[path] = [out];
+        } else if (path.startsWith(`${path}/`)) {
```

Comment:
will this ever be true? 

I imagine you probably want to do something like this:

```
for(const p of Object.keys(paths)) {
    if (p === dep.name) {
        paths[p] = dep.outputs;
    } else if (p.startsWith(`${dep.name}/`)) {
        const nestedPart = p.split(`${dep.name}/`)[1];
        paths[p] = dep.outputs.map(o => `${o}/${nestedPart}`);        
    }
}
```

Reply:
It works as intended, see e2e test. Test should fail if the `else if` is removed.

Reply:
I see, you've changed the implementation again. 🙈

Reply:
path.startsWith(`${path}/`) is always falsy, right? This is incorrect.

Reply:
Ok, fixed it 🙈  Finally i saw the wrong condition.